less/compatibility.less

```diff
@@ -0,0 +1,202 @@
+/*!
+ * Copyright (c) 2020 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Andrii Malytskyi <amalytskyi@tomitribe.com>
+ *   Jonathan Gallimore <jgallimore@tomitribe.com>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+
+.d-flex {
```

Comment:
Would it make more sense to create a class called .display-flex instead of d-flex?

Not sure its super obvious "d" means display here.

Reply:
Also would it be worth it to export that to solstice-assets? There's a few classes here that I think we could add to solstice-assets

Reply:
d is for display, and its a bootstrap 4 > thing. https://getbootstrap.com/docs/5.0/utilities/display/ We're going to be eventually re-writing this entire file, so there isn't really a point for the compatibility file